scipy/stats/tests/test_morestats.py

```diff
@@ -1511,6 +1511,7 @@ def test_array_like(self):
         np.random.seed(54321)
         x = stats.norm.rvs(size=100, loc=0)
         lmbda = 1.5
+        # Should 'stats.yeojohnson(x)' be 'stats.yeojohnson(x, lmbda=lmbda)'?
         xt1, _ = stats.yeojohnson(x)
         xt2, _ = stats.yeojohnson(list(x))
```

Comment:
@NicolasHug Is `lmbda=1.5` meant to be used in this test, or was it just inadvertently left in?  (Added in gh-9305)

Reply:
I think it can be removed, the test is only about testing array-like input

Reply:
Thanks. I've removed it.